src/main/java/mate/academy/hibernate/relations/Main.java

```diff
@@ -16,17 +19,17 @@ public static void main(String[] args) {
         SessionFactory sessionFactory = HibernateUtil.getSessionFactory();
 
         Country usa = new Country("USA");
-        CountryService countryService = null; // TODO: initialize this instance
+        CountryService countryService = new CountryServiceImpl();
         countryService.add(usa);
 
         Actor vinDiesel = new Actor("Vin Diesel");
         vinDiesel.setCountry(usa);
-        ActorService actorService = null; // TODO: initialize this instance
+        ActorService actorService = new ActorServiceImpl();
```

Comment:
Similarly to `CountryService`, you should pass the `SessionFactory` to the `ActorServiceImpl` constructor.